RELEASE.md

```diff
@@ -0,0 +1,15 @@
+To create a release, update the version number in `noetbook_shim/__version__.py`, then run the following:
```

Comment:
Probably the Jupyter Releaser will work out of the box with this pure Python package?

Reply:
Done in https://github.com/jupyterlab/notebook_shim/pull/1/commits/6ed02a9e375cf7ef528c04bfbc168f80ec1d67a6